openeo/internal/jupyter.py

```diff
@@ -0,0 +1,70 @@
+from string import Template
+from collections import Mapping
+import json
+
+# These classes are proxies to visualize openEO responses nicely in Jupyter
+# To show the actual list or dict in Jupyter, use repr() or print()
+
+SCRIPT_URL = 'https://cdn.jsdelivr.net/npm/@openeo/vue-components@2.0.0-beta.1/assets/openeo.js'
+
+class JupyterIntegration():
+
+	# Class instance to store whether the script has been inserted into the DOM
+	# TODO: This fails if a page is reloaded in Jupyter. The variable is still True, but the script doesn't exist in the page any longer.
+	script_sent = False
+
+	def __init__(self, component: str, parameters: dict = {}):
+		self.component = component
+		self.parameters = parameters
+
+	def _repr_html_(self):
+		# Output script tag only once, otherwise JS throws errors
+		script = ''
+		if JupyterIntegration.script_sent == False:
+			script = '<script src="{}"></script>'.format(SCRIPT_URL)
+			JupyterIntegration.script_sent = True
+
+		# Construct HTML
+		template = Template("""
```

Comment:
FYI: usage of this `string.Template` pattern is a bit overkill here. The standard `str.format` with named `{}` placeholders is powerful enough for what you want to do here, e.g.

    return """
        {script}
        <openeo-{component}> .... {props} ... </openeo-{component}>
    """.format(script= ..., component=....)

Reply:
Oh, great. Wasn't aware of that. Fixed.